Tools/ardupilotwaf/boards.py

```diff
@@ -290,6 +290,11 @@ def configure_env(self, cfg, env):
         # We always want to use PRI format macros
         cfg.define('__STDC_FORMAT_MACROS', 1)
 
+        if cfg.options.disable_ekf2:
```

Comment:
We really, really need to find a way of doing this configuration more generally, supplanting our existing `defines.h` and avoiding putting configuration stuff into the `HAL` which really ought not be part of a `HAL`.

Pulling two config parameters out for special treatment isn't great (why I didn't do it initially).  *shrug*.

Reply:
it allowed me to test and debug this code